src/lib.rs

```diff
@@ -234,7 +234,7 @@ pub static GRAPH_URL_BETA: &str = "https://graph.microsoft.com/beta";
 
 pub use crate::client::{Graph, GraphClient};
 pub use graph_error::{GraphFailure, GraphResult};
-pub use graph_http::api_impl::{GraphClientConfiguration, ODataQuery};
+pub use graph_http::api_impl::{GraphClientConfiguration, ODataQuery, Proxy};
```

Comment:
This should be under the http mod. You can add it to some of the other reqwest reexports here: https://github.com/sreeise/graph-rs-sdk/blob/7aa756af45fafec43156d798fff1c47c3fb04b9b/src/lib.rs#L258

Reply:
Thanks for point out, now I expose `reqwest::{Proxy, NoProxy}` here.